pkg/server/handlers.go

```diff
@@ -47,6 +48,10 @@ func Validation(clients *clients.Clients) ([]admission.ValidatingAdmissionHandle
 
 		handlers = append(handlers, psact, globalRoles, globalRoleBindings, prtbs, crtbs, roleTemplates, secrets, nodeDriver, projects)
 	}
+
+	clusterrepo := clusterrepo.NewValidator()
```

Comment:
To confirm, since I don't know about the general requirements, and there is no link to an issue: do we want this to run in both local and remote webhooks?

Reply:
Added the Github Issue. 

What are local and remote webhooks? Can you please point me to some docs? Thanks 👍🏽

Reply:
Yes, I want my validations to run on both local and downstream clusters.